### ISSUE TITLE
Ensures role attribute has an appropriate value for the element

### ISSUE THREAD

**Issue by christinach** [pulibrary/orangelight#2804]
Impact:
minor
Found on:
11/8/2021 at 8:18 PM
Found:
Automatically
Issue Tags:
cat.aria
best-practice
Issue Description
Ensures role attribute has an appropriate value for the element

Element location
nav[role="region"]
Element source
<nav class="pagination" role="region" aria-label="pagination links">
To solve this issue, you need to...
Fix the following:
ARIA role region is not allowed for given element

**Comment by sandbergja** [comment]
I was unable to recreate this issue: which page did this appear on @christinach ?

**Comment by christinach** [comment]
@sandbergja I can't find this as well; maybe it was removed or fixed. I'll close the issue.
